utils/callbacks.py

```
@@ -0,0 +1,45 @@
+"""Callback handling utilities"""
+
+
+def send_callback(callback_url: str, job_id: str, status: str, result: dict = None, error: str = None) -> None:
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)